ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/UIConstant.kt

```diff
@@ -3,6 +3,6 @@ package com.revenuecat.purchases.ui.revenuecatui
 import androidx.compose.ui.unit.dp
 
 internal object UIConstant {
-    val defaultHorizontalPadding = 16.dp
-    val defaultButtonVerticalSpacing = 5.dp
+    val defaultHorizontalPadding = 10.dp
+    val defaultVerticalSpacing = 10.dp
```

Comment:
These are now consistent with iOS. @tonidero let me know if you chose those other values for any reason.

Reply:
No, I think I just took a value that looked ok but didn't copied them from iOS. So this is good, thanks!

Reply:
I'll change this

Reply:
Forgot to address this. I'm changing these to 12